src/dmd/root/array.d

```diff
@@ -146,6 +146,35 @@ public:
         dim++;
     }
 
+    /**
+    Search the array for an element and returns its index.
+    Params:
+        el = element to search for
+
+    Returns: index of the element, -1 otherwise
+    */
+    ptrdiff_t indexOf(T el) nothrow
+    {
+        foreach (i; 0 .. dim)
```

Comment:
Why not a regular `foreach` instead of a `foreach` range loop?

Reply:
No reason. Thanks for the reminder!